packages/core/src/types/ProgressiveListener.ts

```diff
@@ -0,0 +1,17 @@
+export type ProgressiveListener = {
+  /** Called when an image is loaded.  May be called multiple times with increasing
```

Comment:
this will not grabbed from our api generator, the api generator for website for documentation requires explicit /** then new line